File: beginner/javascript/1180.js
var input = require('fs').readFileSync('stdin', 'utf8');
var linhas = input.split('\n');

var n = parseInt(linhas.shift());
var v = linhas.shift().split(" ").map(valor => parseInt(valor));

var menor = v[0];
var posicao = 0;

for (var i = 0; i < n; i++) {
    if (v[i] < menor) {
        menor = v[i];
        posicao = i;
    }
}

console.log(`Menor valor: ${menor}`);
console.log(`Posicao: ${posicao}`);

File: beginner/python/1180.py
n = int(input())
v = list(map(int, input().split()))

menor = v[0]
posicao = 0

for i in range(0,n):
    if v[i] < menor:
        menor = v[i]
        posicao = i
    
print("Menor valor: %d" % menor)
print("Posicao: %d" % posicao)
